### PR TITLE
Fix for Firefox WebVR

### PR DESCRIPTION
**Pull request by brianpeiris**
This fixes VRControls to workaround the behaviour in the new Firefox Nightly builds where it [returns multiple position sensor devices](https://mail.mozilla.org/pipermail/web-vr-discuss/2015-April/000611.html) (one for Oculus and one for Cardboard) regardless of whether you're on a desktop PC or not. 

I've also tweaked the VR Cubes example so that it makes a bit more sense for VR.

### CODE REVIEW

**Comment by dmarcos** [comment]
I would prefer to come up with a proper API to handle multiple VR inputs. This patch basically reverts what Boris had done in https://github.com/mrdoob/three.js/pull/6286

**Comment by brianpeiris** [comment]
I've added a commit to allow multiple sensors again and instead filter out the Cardboard sensor if the Rift is detected. 

This supports the headtrackr polyfill example mentioned in #6286. 

Of course, I assume that deviceId is a reliable constant, which seems to be the case according to vvuk/gecko-dev/@104c1f0df2b0a73119209736ade2e37289433f77. I could have used deviceName instead but that might not be a good idea. I also assume that there isn't a phantom Oculus device in Firefox for Android!

**Comment by brianpeiris** [comment]
Alright, turns out deviceId is not constant across browser sessions, so I've changed it to use deviceName instead, as @vvuk mentioned in #6286.

**Comment by dmarcos** [comment]
Discarding the cardboard if the Rift is present feels very hacky and not worth the extra complexity.  We're piling hacks on top of hacks. 

In the short term I would prefer to revert https://github.com/mrdoob/three.js/pull/6286 and get Firefox Nightly working again. We can then put more thought on how to properly handle multiple input devices. Brian, if you are interested feel free to come up with an API proposal that you feel right so we can discuss it.

**Comment by brianpeiris** [comment]
Fair enough, it does feel hacky. I'm not particularly interested in getting multiple devices working at the moment. I'll leave that to @borismus. I just wanted to get Firefox Nightly working, so I'm ok with reverting #6286. 

I'll open a new PR for the VR Cubes changes.

File: examples/vr_cubes.html [hunk 1]
```diff
@@ -81,6 +82,16 @@
 
 				scene = new THREE.Scene();
 
+				crosshair = new THREE.Mesh(
+					new THREE.RingGeometry( 0.5, 1, 32 ),
+					new THREE.LineBasicMaterial( {
```
Comment (mrdoob) [inline examples/vr_cubes.html]: This should be `MeshBasicMaterial`.

Comment (brianpeiris) [inline examples/vr_cubes.html]: Fixed. Thanks.

**Comment by mrdoob** [comment]
Thanks!

**Comment by dmarcos** [comment]
Thanks guys! It works now on Firefox Nightly

**Comment by brianpeiris** [comment]
Sweet!

**Pull request merged**